phpunit/tests/fonts/font-library/fontDeleteFilesFromWpContentFolder.php

```diff
@@ -0,0 +1,160 @@
+<?php
+// @core-merge: Do not include these tests, they are for Gutenberg only.
+
+/**
+ * Test deleting font files from the previous fonts folder (wp-content/fonts).
+ *
+ * @package WordPress
+ * @subpackage Font Library
+ *
+ * @group fonts
+ * @group font-library
+ *
+ * @covers ::gutenberg_before_delete_font_face
+ */
+class Tests_Font_Delete_Files_From_Wp_Content_Folder extends WP_UnitTestCase {
```

Comment:
Are you able to add docblocks for the methods and properties throughout?

Reply:
👍🏼 done in https://github.com/WordPress/gutenberg/pull/60354/commits/abbbb28da3a87cd5edd39d6b20791e2a33ae2cc2